include/rrc/io/IOError.h

```diff
@@ -0,0 +1,26 @@
+/**
+ *  @autor nikitas
```

Comment:
Sometimes you're nikitas, sometimes you're Nikita Somenkov. You need to define yourself.